docs/02-getting-started/02-installation.md

```diff
@@ -26,6 +26,32 @@ In order to deploy to AWS, you will also need:
 
 ## Wing CLI
 
+:::caution If you were in our closed alpha
```

Comment:
```suggestion
<details>
  <summary>caution If you were in our closed alpha</summary>
```

Reply:
I think it's better to make this very visible for now, and we'll remove it soon enough.